app/facades/landmarks_show_facade.rb

```diff
@@ -0,0 +1,15 @@
+class LandmarksShowFacade
+  attr_reader :id
+
+  def initialize(id)
+    @id = id
+  end
+
+  def landmark
+    @_landmark ||= Landmark.find(@id)
+  end
+
+  def picture
+    "https://maps.googleapis.com/maps/api/place/photo?maxheight=400&maxwidth=600&key=#{ENV['GOOGLE_PLACES_API_KEY']}&photoreference=#{landmark.photo_reference}"
```

Comment:
These aren't working for me -- I get a huge default image for several different places

Reply:
Putting the Google API key in the application.yml and also updating the landmark table should fix this issue.

Reply:
Try reloading landmarks table.